examples/stable_diffusion_v2/dreambooth_finetune.md

```diff
@@ -15,7 +15,7 @@ The following picture gives a high-level overview of the DreamBooth Method.
 </p>
 
 
-DreamBooth only requires a pretrained Text-to-Image model and a few (3-5) subject's images along with its name as a Unique Identifier. Unique Identifier prevents the Text-to-Image model from **language drift** [<a href="#references">2</a>]. Language Drift is a phenonomon that the model tends to associate the class name (e.g., "dog") with the specific instance (e.g., your dog). Because during finetuning, the model only sees the prompt "dog" and your dog's pictures, so that it gradually forgets other dogs' look. Therefore, we need a Unique Identifier ("sks") to differentiate your a dog ("sks dog") and a general dog ("dog").
+DreamBooth only requires a pretrained Text-to-Image model and a few (3-5) subject's images (more images are needed for complex objects) along with its name as a Unique Identifier. Unique Identifier prevents the Text-to-Image model from **language drift** [<a href="#references">2</a>]. Language Drift is a phenonomon that the model tends to associate the class name (e.g., "dog") with the specific instance (e.g., your dog). Because during finetuning, the model only sees the prompt "dog" and your dog's pictures, so that it gradually forgets other dogs' look. Therefore, we need a Unique Identifier ("sks") to differentiate your a dog ("sks dog") and a general dog ("dog").
```

Comment:
maybe be consistent with using text-to-image or Text-to-Image;
a few subjects'; with a unique id for each name (or use gpt I guess);
your a dog -> your dog

Reply:
fixed.